openbas-api/src/main/java/io/openbas/rest/scenario/ScenarioApi.java

```diff
@@ -286,4 +286,14 @@ public Exercise createRunningExerciseFromScenario(
     return scenarioToExerciseService.toExercise(
         scenario, now().truncatedTo(MINUTES).plus(1, MINUTES), true);
   }
+
+  @PostMapping(SCENARIO_URI + "/{scenarioId}/check-rules")
```

Comment:
```suggestion
  @PostMapping(SCENARIO_URI + "/{scenarioId}/check-rules")
  @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Returns whether or not the rules apply")})
  @Operation(summary = "Check rules", description = "Check if the rules apply to this scenario")
```
Would be nice to have the swagger docs on this new endpoint :)

Reply:
Documentation added for all the new apis + dto in the new commit